macros/SectionOnPage.ejs

```diff
@@ -5,11 +5,30 @@
 //
 // $0   Page path
 // $1   Section name
+// $2   Element name to wrap the page title (not the section name) in;
+//      leave blank or don't specify this parameter to not wrap the title.
+//      Don't include the angle brackets! Only specific tags are permitted;
+//      they're in the allowedWrappers array.
```

Comment:
Thanks for adding docs about this parameter. I think it is also a good idea to add a sample call:
```
// Example call
// {{SectionOnPage("/en-US/docs/Web/CSS/color_value", "Color keywords", "code")}}
```